### PR TITLE
Mangasee: Add intent filter

### PR DESCRIPTION
**Pull request by suhan-paradkar**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/nepnep/mangasee/src/MangaseeUrlActivity.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import kotlin.system.exitProcess
+ 
+/**
+ * Springboard that accepts https://mangasee123.com/manga/xxxxxx intents and redirects them to
+ * the main Tachiyomi process.
+ */
+class IMHentaiUrlActivity : Activity() {
```
Comment (alessandrojean) [inline multisrc/overrides/nepnep/mangasee/src/MangaseeUrlActivity.kt:14]:
```suggestion
class MangaseeUrlActivity : Activity() {
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]
It seems you haven't added the `id:` handling on the base class, so it will not work.

Marking it as draft for now.

File: multisrc/overrides/nepnep/mangasee/src/MangaseeUrlActivity.kt (hunk 1 above)
Comment (suhan-paradkar) [reply multisrc/overrides/nepnep/mangasee/src/MangaseeUrlActivity.kt]: Done!!

**Comment by suhan-paradkar** [comment]
> It seems you haven't added the `id:` handling on the base class, so it will not work.
> 
> Marking it as draft for now.

Well..  I added it to the base class (I mean in the overrides)....

File: multisrc/overrides/nepnep/mangasee/src/MangaSee.kt [hunk 2]
```diff
@@ -15,4 +15,19 @@ class MangaSee : NepNep("MangaSee", "https://mangasee123.com", "en") {
...
+                .asObservableSuccess() 
+                .map { 
+                response ->
+                    val manga = mangaDetailsParse(response) 
+                    manga.url = "/manga/$id/" 
+                    MangasPage(listOf(manga), false)
+                }
+        }
+        return super.fetchSearchManga(page, query, filters)
+    }
```
Comment (alessandrojean) [inline multisrc/overrides/nepnep/mangasee/src/MangaSee.kt:32]: Probably added a `}` as a mistake, so I fixed it. Check if it still works as expected.

```suggestion
override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
        if (query.startsWith("id:")) {
            val id = query.substringAfter("id:")
            return client.newCall(GET("$baseUrl/manga/$id/"))
                .asObservableSuccess() 
                .map { response ->
                    val manga = mangaDetailsParse(response) 
                    manga.url = "/manga/$id/" 
                    MangasPage(listOf(manga), false)
                }
        }
        
        return super.fetchSearchManga(page, query, filters)
    }
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Review by alessandrojean — APPROVED** [review]

File: multisrc/overrides/nepnep/mangasee/src/MangaSee.kt [hunk 3]
```diff
@@ -15,4 +15,19 @@ class MangaSee : NepNep("MangaSee", "https://mangasee123.com", "en") {
         .readTimeout(1, TimeUnit.MINUTES)
         .writeTimeout(1, TimeUnit.MINUTES)
         .build()
+
+override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
```
Comment (alessandrojean) [inline multisrc/overrides/nepnep/mangasee/src/MangaSee.kt:19]:
```suggestion
    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
```

**Comment by alessandrojean** [comment]
You need to fix some linting issues related to trailing spaces, otherwise the build will keep failing.

As a reference, they're listed below:

```
MangaseeUrlActivity.kt:2:1: Lint error > [no-trailing-spaces] Trailing space(s)
MangaseeUrlActivity.kt:9:1: Lint error > [no-trailing-spaces] Trailing space(s)
MangaseeUrlActivity.kt:25:1: Lint error > [no-trailing-spaces] Trailing space(s)
MangaseeUrlActivity.kt:34:1: Lint error > [no-trailing-spaces] Trailing space(s)
MangaseeUrlActivity.kt:39:1: Lint error > [no-trailing-spaces] Trailing space(s)
MangaseeUrlActivity.kt:40:1: Lint error > [no-consecutive-blank-lines] Needless blank line(s)
MangaSee.kt:23:39: Lint error > [no-trailing-spaces] Trailing space(s)
MangaSee.kt:25:60: Lint error > [no-trailing-spaces] Trailing space(s)
MangaSee.kt:26:46: Lint error > [no-trailing-spaces] Trailing space(s)
MangaSee.kt:30:1: Lint error > [no-trailing-spaces] Trailing space(s)
```

**Comment by stevenyomi** [comment]
Just select `NepNepGenerator` config and run it once. It will format the code correctly.

**Comment by suhan-paradkar** [comment]
I guess it must pass CI by now......  I had it run through ktlint...

**Pull request merged**